idd/Energy+.idd.in

```diff
@@ -24823,6 +24823,55 @@ AirflowNetwork:SimulationControl,
       \key Yes
       \key No
       \default No
+ A10, \field Duct Sizing Method
+      \type choice
+      \key None
+      \key MaximumVelocity
+      \key PressureLoss
+      \key PressureLossWithMaximumVelocity
+      \default None
+ N7 , \field Duct Sizing Factor
```

Comment:
@mjwitte Here is what I think shown below. Please let me know what you think.

! Basic parameters
AirflowNetwork:SimulationControl,
      \min-fields 16
      \unique-object
      \memo This object defines the global parameters used in an Airflow Network simulation.
...
 A10, \field Duct Sizing Method
      \type choice
      \key None
      \key MaximumVelocity
      \key PressureLoss
      \key PressureLossWithMaximumVelocity
      \default None
 A11, \field Duct Sizing Parameter Name

AirflowNetwork:DuctSizing:Parameter,
 A1 , \field Name
      \required-field
      \note Enter a unique name for this object.
 N1 , \field Duct Sizing Factor
      \type real
      \minimum> 0.0
      \default 1.0
 N2 , \field Maximum Airflow Velocity
      \type real
      \units m/s
      \minimum >0.0
      \maximum 25.0
      \default 5.0
      \note Used only if Duct Sizing Type = MaximumVelocity or PressureLossWithMaximumVelocity.
      \note When MaximumVelocity is entered, duct diameter is calculated at D = flow rate / 
      \note cross section area.
      \note When PressureLossWithMaximumVelocity is entered, duct diameter is calculated based on 
      \note PressureLoss. The value is used to check to ensure the final velocity is less than
      \note the maximum value. If greater, final value will be obtained from MaximumVelocity.
      \note This field is apply for trunk size, while branch size is based on total pressure drop.
 N3 , \field Total Pressure Loss Across Supply Trunk
      \type real
      \units Pa
      \minimum >0.0
      \note Used only if Duct Sizing Type = PressureLoss or PressureLossWithMaximumVelocity.
      \note When PressureLoss is entered, duct diameter is calculated using Colebrook's equation  
      \note When PressureLossWithMaximumVelocity is entered, duct diameter is calculated based on 
      \note PressureLoss. The value is used to check to ensure the final velocity is less than
      \note the maximum value. If greater, final value will be obtained from MaximumVelocity.
      \note This field is apply for trunk size, while branch size is based on total pressure drop.
 N4 , \field Total Pressure Loss Across Supply Branch
      \type real
      \units Pa
      \minimum >0.0
      \note Duct diameter is calculated using Colebrook's equation  
 N5, \field Total Pressure Loss Across Return Trunk
      \type real
      \units Pa
      \minimum >0.0
      \note Duct diameter is calculated using Colebrook's equation  
 N6; \field Total Pressure Loss Across Return Branch
      \type real
      \units Pa
      \minimum >0.0
      \note Duct diameter is calculated using Colebrook's equation

Reply:
@mjwitte Any comments for the proposed IDD change based on your previous comments?

Reply:
@lgu1234 To be consistent with other AFN objects, I would call the new object AirflowNetwork:Distribution:DuctSizing. Perhaps the Duct Sizing Method field belongs here as well?  @jasondegraw Where would you put these new fields?

Reply:
@mjwitte @jasondegraw I have a solution to support "Duct Sizing Method" inside the new object. However, I will remove option of None for the "Duct Sizing Method". If a user would like to turn off the Duct Sizing, it is easy to remove  the new object name in the last field of AirflowNework:SimulationControl.

Reply:
@lgu1234 That seems overly complicated. Perhaps we should structure this more the way overall EnergyPlus sizing is done.

In AirflowNetwork:SimulationControl, new field:
```
A10,  \field Do Distribution Duct Sizing Calculation
      \ Controls duct sizing. See  AirflowNetwork:Distribution:DuctSizing for sizing options.
      \type choice
      \key Yes
      \key No
      \default No
```
New object:
```
AirflowNetwork:Distribution:DuctSizing,
A1, \field Name (just because every object needs a name; no need to reference it in the SimulationControl object)
A2, \field Duct Sizing Method
\type choice
\key MaximumVelocity
\key PressureLoss
\key PressureLossWithMaximumVelocity
\default ????
...
```